### PR TITLE
WINUI-3059 : Updated WinUI UG content document with gauges' new features

### PR DESCRIPTION
**Pull request by VishalOmprasad**
## Description ###
Updated WinUI linear and radial gauges shape pointer UG content related to new features (Shadow and Overlay support).

## Output screenshots ##
* Linear gauge shape pointer shadow support
image

* Radial gauge shape pointer shadow support
image

* radial gauge shape pointer overlay support
image

### CODE REVIEW

**Review by vigneshrameshsync — COMMENTED** [review]
The content is fine.

File: winui/Linear-Gauge/shape-pointer.md [hunk 1]
```diff
@@ -159,6 +159,46 @@ this.Content = sfLinearGauge;
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change shape pointer elevation](images/shape-pointer/pointer_shadow.png)
+
 ## Shape position customization
 
 The shape pointer can be moved near or far from its actual position in X or Y direction using the [`OffsetPoint`](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Gauges.LinearMarkerPointer.html#Syncfusion_UI_Xaml_Gauges_LinearMarkerPointer_OffsetPoint) property.
```
Comment (OnyimbiNelson) [inline winui/Linear-Gauge/shape-pointer.md:204]: The shape pointer can be moved near or far from its actual position in the X or Y direction using the [`OffsetPoint`](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Gauges.LinearMarkerPointer.html#Syncfusion_UI_Xaml_Gauges_LinearMarkerPointer_OffsetPoint) property.

File: winui/Linear-Gauge/shape-pointer.md [hunk 2]
```diff
@@ -159,6 +159,46 @@ this.Content = sfLinearGauge;
 
 ![shape pointer customization](images/shape-pointer/shape_customization.png)
 
+## Shadow support
+
+The shadow can be applied by the `HasShadow` property.
```
Comment (OnyimbiNelson) [inline winui/Linear-Gauge/shape-pointer.md:164]: The shadow can be applied by using the `HasShadow` property.

File: winui/Radial-Gauge/shape-pointer.md [hunk 3]
```diff
@@ -113,6 +113,99 @@ this.Content = sfRadialGauge;
 
 ![WinUI Radial Gauge custom shape pointer](images/marker-pointers/winui-radial-gauge-custom-marker-pointer.png)
 
+## Shadow support
+
+The shadow can be applied by the `HasShadow` property.
```
Comment (OnyimbiNelson) [inline winui/Radial-Gauge/shape-pointer.md:118]: The shadow can be applied by using the `HasShadow` property.

File: winui/Linear-Gauge/shape-pointer.md (hunk 1 above)
Comment (VishalOmprasad) [reply winui/Linear-Gauge/shape-pointer.md]: Included "the" in the content.

File: winui/Linear-Gauge/shape-pointer.md (hunk 2 above)
Comment (VishalOmprasad) [reply winui/Linear-Gauge/shape-pointer.md]: Changed the content as you mentioned.

File: winui/Radial-Gauge/shape-pointer.md (hunk 3 above)
Comment (VishalOmprasad) [reply winui/Radial-Gauge/shape-pointer.md]: Changed the content as you mentioned.

**Review by vigneshrameshsync — APPROVED** [review]
Fine

**Pull request merged**